code-challenges.test.js

```diff
@@ -26,17 +24,52 @@ var num2 = 0
 var num3 = -7
 // Expected output: "-7 is not divisble by three"
 
+//Write function named divideThree with the parameter/argument named num
 
+const divideThree = (num) => {
```

Comment:
This is well done. you function is well pseudo coded and I can follow its logic clearly.